### PR TITLE
feat: added reference to email exposure through the NPM API

### PR DESCRIPTION
**Pull request by UlisesGascon**
### TL;DR
The current [NPM API](https://registry.npmjs.org) is exposing under the `maintainers` and `_npmUser` properties the email addresses of the package maintainers and the author of the release publication.

### Example
Here is an example of it with [Express@4.18.1](https://github.com/expressjs/express/blob/master/package.json):
- The current `package.json` includes some email addresses for `author` and `contributors`. [See](https://github.com/expressjs/express/blob/master/package.json)
- The API also includes the `maintainers` and `_npmUser` information that is not included in the `package.json`. [see](https://registry.npmjs.org/express/4.18.1)

These extra fields were not part of the API in the past (for example [Express@2.5.0](https://registry.npmjs.org/express/2.5.0)) and most npm users are not aware of this change.

The same exposure applies if the package does not include any email in the `author` or `contributors` in the `package.json`.

Currently NPM does not include any way to prevent that. In the past we had a similar situation with Github commits authority, but they provides now the option to use a privacy version like `<username>@users.noreply.github.com` as the email address. [See](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)

### CODE REVIEW

**Comment by ljharb** [comment]
npm has always, since day one, exposed every publisher’s email address via the API - `npm show qs` for example shows the publisher’s email address (me, in this case).

It’s not a change - you waive privacy on your email address when you have the ability to publish something.

Your PR text is accurate, but there’s no risk in exposing an email address - most people do so via github anyways - so I’m not sure why it’s worth remarking on.

**Comment by UlisesGascon** [comment]
Thanks for the feedback @ljharb! I will close the PR
